# Unit 5: **2-nd law of thermodynamics: Entropy, Helmholtz, and Gibbs free energies. Spontaneous processes and their directionality. 3-rd law of thermodynamics: Absolute entropies.**

# 1. Theory

# 2. Examples

## Problem 1 [Eremin Example 4-1]

Determine the dependence of entropy on volume for a thermodynamic system that described by the equation of the state (for one mole)

$(p + \frac{a}{V^2})(V - b) = RT$

### Solution

Solve for P:

$P = \frac{RT}{V-b} - \frac{a}{V^2}$

Compute the derivative:

$(\frac{\partial P}{\partial T})_V = \frac{R}{V-b}$

The entropy change is computed as:

$\Delta S = \int_{V_1}^{V_2} (\frac{\partial P}{\partial T})_V dV = \int_{V_1}^{V_2} \frac{R}{V-b} dV = R ln(\frac{V_2-b}{V_1-b}) $

## Problem 22 [Eremin Example 4-2]

Calculate the change in entropy when 0.7 mole of monoclinic sulfur is heated from 25 to 200 Celsius at 1 atm pressure. Molar heat capacity of sulfur is given by:

$C_{p,m}(S(s)) = 23.64$ &nbsp; $J \times mol^{-1}K^{-1}$,

$C_{p,m}(S(l)) = 35.73 + 1.17 \times 10^{-3} T$ &nbsp; $J \times mol^{-1}K^{-1}$

The melting temperature of monoclinic sulfur is 119 Celsius, the specific heat of melting 45.2 J/g

### Solution

The total change of entropy $\Delta S$ is composed on entropy changes of three processes:

* heating the solid from $T_1 = 25$ Celsius to the melting point $T_2 = 119$ Celsius, $\Delta S_1 = n \int_{T_1}^{T_2} \frac{C_{p,m}(s)}{T} dT$

because the solid sulfur's heat capacity is constants, the integral becomes:

$n \int_{T_1}^{T_2} \frac{C_{p,m}(s)}{T} dT = n C_{p,m}(s) ln(\frac{T_2}{T_1})$

* melting the solid sulfur at $T_2 = 119 Celsius$

$\Delta S_2 = m \frac{Q_{s}}{T_2} $

* heating the liquid from the melting point $T_2 = 119$ Celsius to the desired temperature $T_3 = 200$ Celsius

In this case, heat capacity depends on temperature: $C_{p,m}(l) = a + b T$ so

$\Delta S_3 = n \int_{T_2}^{T_3} \frac{C_{p,m}(l)}{T} dT = a ln(\frac{T_3}{T_2}) + b(T_3 - T_2)$

In [2]:
import math 

n = 0.7 # moles
T1 = 25 + 273.15 # K
T2 = 119 + 273.15 # K melting temperature
T3 = 200 + 273.15 # K
Cpm_solid = 23.64 # J / mol * K 
Q_melt = 45.2 # J/g  melting heat 
m = 32.0 * n # molar mass of sulfur

a = 35.73 
b = 1.17e-3 

dS1 = n * Cpm_solid * math.log(T2 / T1)
dH = m * Q_melt 

dS2 = dH / T2 

dS3 = n * ( a * math.log(T3/T2) + b * (T3 - T2) )

dS = dS1 + dS2 + dS3

print(F"dS = {dS} J/K")

dS = 11.879416120850149 J/K


## Problem 3 [Eremin Example 4-3]

Calculate the change in entropy of 1000 g of water as a result of its freezing at -5 Celsius. The heat of ice melting at 0 Celsius is equal to 6008 J/mol. The heat capacities of ice and water are 34.7 and 75.3 J/(mol  K), respectively. Explain why the process of freezing is spontanous even though the entropy of the systems decreases when freezing.

### Solution

## Problem 4

(5 points max) Compute $\Delta S$ for heating 0.4 mole of NaCl from 20℃ to 850℃. The molar heat capacity of NaCl in solid and liquid states is given by: 

$C_{p,m}(NaCl,solid)=45.94 + 16.32*10^{-3} T  \frac{J}{(mol*K}$

$C_{p,m}(NaCl,liquid)=66.53  \frac{J}{mol*K}$

NaCl melts at 800 ℃ with the molar fusion heat of $\Delta_{fus} H = 31.0 kJ/mol$

### Solution

The process consists of 3 steps - heating up the solid until the fusion temperature, melting, and then heating the liquid phase up to the target temperature.

The first process:

$\Delta S_1 = n \int_{T_1}^{T_2} \frac{C_{p,m}(solid)}{T} dT = n \int_{T_1}^{T_2} \frac{45.94 + 16.32*10^{-3} T}{T} dT = 45.94 ln(\frac{T_2}{T_1}) + 16.32*10^{-3} * (T_2 - T_1)$

The second process:

$\Delta S_2 = n \frac{\Delta_{fus} H}{T_{fus}} = n \frac{\Delta_{fus} H}{T_{2}}$

The third process:

$\Delta S_3 = n \int_{T_2}^{T_3} \frac{C_{p,m}(liquid)}{T} dT = n \int_{T_2}^{T_3} \frac{66.53}{T} dT = 66.53 ln(\frac{T_3}{T_2})$

In [3]:
T1 = 20 + 273.15
T2 = 800 + 273.15
T3 = 850 + 273.15
n = 0.4 # moles of NaCl

dH_fus = 31.0e+3 # J/mol

dS1 = n * (45.94 * math.log(T2/T1) + 16.32e-3 * (T2 - T1) )

dS2 = n* dH_fus / T2

dS3 = n * 66.53 * math.log(T3/T2)

dS = dS1 + dS2 + dS3

print(F" dS = {dS} J/K")

 dS = 41.704459703949226 J/K


## Problem 5

(5 points max). An iron (Fe) block of mass 0.5 kg heated to temperature t=100℃ is immersed into 1 kg of water at t=0℃. Compute $\Delta S_{Fe}$, $\Delta S_{water}$, and the $\Delta S_{tot}$ (where “tot” means the Fe block + water), if the specific heat capacities of iron and water are $C_{Fe} =0.447 \frac{J}{g K}$ and $C_{water} = 4.184 \frac{J}{g K}$ respectively. We assume that the iron + water system is isolated. Does your result agree with an expectation based on the second law of thermodynamics. What is that expectation? Explain. 

### Solution

First, we need to compute determine the amount of heat transfered between components (Fe block and water).

Heat transferred from Fe block will be:

$|q_{Fe}| = m_{Fe} C_{Fe} (T_{Fe} - T_{eq})$

where $T_{Fe} = 100 + 273.15 K$ and $T_{eq}$ is yet unknown temperature at equilibrium.

Likewise, heat transferred to water will be:

$|q_{water}| = m_{water} C_{water} (T_{eq} - T_{water})$

At equilibrium,

$|q_{Fe}| = |q_{water}|$

So:

$m_{Fe} C_{Fe} (T_{Fe} - T_{eq}) = m_{water} C_{water} (T_{eq} - T_{water})$

So:

$T_{eq} = \frac{m_{Fe} C_{Fe} T_{Fe} + m_{water} C_{water} T_{water}}{m_{Fe} C_{Fe} + m_{water} C_{water}} $

In [4]:
T_Fe = 100 + 273.15 # K
T_water = 0.0 + 273.15 # K
C_Fe = 0.447 # J/g*K
C_water = 4.184 # J/g*K
m_Fe = 0.5 * 1000.0 # g
m_water = 1.0 * 1000 # g

# Compute the equilibrium temperrature
T_eq = (m_Fe * C_Fe * T_Fe + m_water * C_water * T_water)/(m_Fe * C_Fe + m_water * C_water)

print(F" equilibrium temperature = {T_eq - 273.15} Celsius")

# Now compute the amount of heat transferred:
q_water = m_water * C_water * (T_eq - T_water)
print(F" heat trasferred to water = {q_water} J")

q_Fe = -q_water
print(F" heat trasferred from Fe = {q_Fe} J")


dS_Fe = q_Fe/T_Fe
print(F" dS for Fe = {dS_Fe} J/K")

dS_water = q_water/T_water
print(F" dS for water = {dS_water} J/K")

dS_tot = dS_Fe + dS_water
print(F" dS total = {dS_tot} J/K")

 equilibrium temperature = 5.070901871809383 Celsius
 heat trasferred to water = 21216.65343165046 J
 heat trasferred from Fe = -21216.65343165046 J
 dS for Fe = -56.8582431506109 J/K
 dS for water = 77.67400121416972 J/K
 dS total = 20.815758063558818 J/K


### Answers 

$\Delta S_{Fe} = -56.9 $ J/K

$\Delta S_{water} = 77.7 $ J/K

$\Delta S_{tot} = 20.8 $ J/K

The overall change of entropy is positive, as expected from the second law of TD for an isolated system. So, the heat transfer from Fe to water occurs spontaneously. 

# 3. Problems

## 3.1. [Eremin 4-1]

Give an example of thermodynamic process, which can be carried out as a reversible and irreversible. Calculate Changes in the entropy of the system and the environment in in both cases.

## 3.2. [Eremin 4-3]

Calculate entropy of a mole of neon at 500 K if at 298 K and the same volume, neon's entropy is 146.2 J/(mol. K).

## 3.3. [Eremin 4-4]

Calculate the entropy change when heating 11.2 L of nitrogen from 0 to 50 °C and simultaneously reducing pressure from 1 atm to 0.01 Atm. 

## 3.4. [Eremin 4-5]

One mole of helium at 100 °C and 1 atm is mixed with 0.5 mol of neon at 0 °C and 1 atm. Determine the change in entropy if the finite pressure is 1 atm.

## 3.5. [Eremin 4-6]

Calculate the entropy change in the result of formation of $1 m^3$ of air from nitrogen and oxygen (0.2 mole fraction) at a temperature of 25 °C and pressure of 1 atm.

## 3.6. [Eremin 4-7]

Three moles of an ideal monatomic gas (C_V = 3.0 cal/(mol K)) with initial temperature and pressure of $T_1 = 350 K$ and $P_1 = 5.0 atm$ are expanded reversible and adiabatically to a pressure of $P_2 = 1.0 atm$. Calculate the final temperature, volume, the work performed on the system, as well as the changes of internal energy, enthalpy and entropy in this process.

## 3.7. [Eremin 4-8]

Calculate the entropy change when 0.4 mol of sodium chloride are heated from 20 to 850 °C. The molar heat capacity of sodium chloride is given by:

$C_{p,m}(NaCl(s)) = 45.94 + 16.32 \times 10^{-3}T$ &nbsp; $J mol^{-1} K^{-1}$

$C_{p,m}(NaCl(l)) = 66.53 J mol^{-1} K^{-1}$

The melting point of sodium chloride is 800 °C, The heat of melting is 31.0 kJ/mol.

## 3.8 [Eremin 4-9]

Calculate the entropy change when 5 kg of water at 80 °C are mixed with 10 kg of water at 20 °C. The specific heat capacity of water should be taken as follows: $C_p(H_2O) = 4.184 J g^{-1} K^{-1}$.


## 3.9. [Eremin 4-10]

Calculate the entropy change when 200 g of ice at the temperature of 0 °C is added to 200 g of water at  90 °C in an isolated vessel. Heat of melting of ice is 6.0 kJ/mol.

## 3.10. [Eremin 4-11]

For a certain solid body the expansion coefficient was found to depend on pressure as $(\frac{\partial V}{\partial T})_P = a + b P + c P^2$ in the pressure interval between $P_1$ and $P_2$. How much will the entropy of this body decrease when the system is compressed from pressure $P_1$ to pressure $P_2$?

## 3.11. [Eremin 4-12]

Find the entropy change of n moles of an idea gas if it expand isothermally from pressure $P_1$ to pressure $P_2$: a) reversible; b) against external pressure $P < P_2$.

## 3.12. [Eremin 4-13]

Write down the expression for calculation of absolute entropy of one mole of water at temperature of 300 Celsius and pressure of 2 atm.

## 3.13. [Eremin 4-14]

Draw a graph of the standard entropy of water from temperature in the range from 0 to 400 K

## 3.14. [Eremin 4-15]

Write down the expression for the entropy of one mole of an ideal gas as a function of temperature and pressure (heat capacity is considered constant).

## 3.15. [Eremin 4-16]

Determine the dependence of entropy on volume for a thermodynamic system that is described by the equation of state (for one mole):

$(P + \frac{a}{TV^2})(V-b) = R T$

## 3.16. [Eremin 4-17]

Determine the dependence of entropy on volume for a thermodynamic system that is described by the equation of state (for one mole):

$PV = RT (1 + \frac{b}{V} + \frac{c}{V^2})$



## 3.17. [Eremin 4-18]

One mole of gas is described by the following equation of state: $(P + f(V))(V-b) = RT$, where $f(V)$ is some function that does not depends on the temperature. Calculate the change entropy of the gas when it undergoes an irreversible isothermal expansion from volume $V_1$ to volume $V_2$.



## 3.18. [Eremin 4-19]

Calculate the entropy change of 1000 g methanol as a result of its freezing at -105 °C. Heat of melting of solid methanol at -98 °C (melting temperature) is 3160 J/mol. Heat capacity of solid and liquid methanol are equal to 55.6 and 81.6 J/(mol. K), respectively. Explain why entropy at freezing decreases, although the process of spontaneous.

# 4. Answers
